app/com/linkedin/drelephant/exceptions/ExceptionFinder.java

```diff
@@ -0,0 +1,167 @@
+package com.linkedin.drelephant.exceptions;
```

Comment:
Can I add this to to do list? I am not sure whether I'll be able to do it by tomorrow.